vignettes/hazardItems.Rmd

```diff
@@ -34,6 +35,22 @@ This package **hazardItems** provides services in support of coastal change haza
 
 `template` service creates or replaces new child items based on an input id. 
 
+`setBaseURL` and `authenticateUser` are supporting functions for directing requests and posts to the proper development, testing or production tier, and checking proper access for posting data.
+
+
+
+#Installation
+To install **hazardItems** 
+
+Install required packages:
+```{r, eval=FALSE}
+install.packages(c("httr","scales","rjson","png","maps","mapdata","XML","devtools","roxygen2","knitr","rmarkdown","mapdata"))
+```
+
+Then:
+```{r, eval=FALSE}
```

Comment:
At this point, I don't this `hazardItems` is actually installed, just all of its dependencies. but...it isn't up on GRAN (owi.usgs.gov/R) yet, so we can leave those instructions as a to do.